driver-adapters/planetscale-node-basic/prisma/schema.prisma

```diff
@@ -0,0 +1,16 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mysql"
+  // fake url to ensure the driver adapter is used
+  url      = env("DRIVER_ADAPTERS_PLANETSCALE_NODE_BASIC_DATABASE_URL")
```

Comment:
This doesn't look like a fake url

Reply:
https://prisma-company.slack.com/archives/C04TW4A2H8C/p1693593638065299

Reply:
Maybe, at least, update the comment?

Reply:
Done